File: addons/l10n_cl/models/res_country.py
from odoo import fields, models


class ResCountry(models.Model):
    _inherit = "res.country"

    l10n_cl_customs_code = fields.Char(string="Customs Code")
    l10n_cl_customs_name = fields.Char(string="Customs Name")
    l10n_cl_customs_abbreviation = fields.Char(string="Customs Abbreviation")

File: addons/web/static/tests/_framework/mock_server/mock_models/res_country.js
import { ServerModel } from "../mock_model";
import { serverState } from "../../mock_server_state.hoot";

export class ResCountry extends ServerModel {
    _name = "res.country";

    _records = serverState.countries.map((country) => ({
        id: country.id,
        name: country.name,
        code: country.code,
    }));
}
